platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/LdapClaimsHandler.java

```diff
@@ -87,20 +98,59 @@ public void setPropertyFileLocation(String propertyFileLocation) {
     this.propertyFileLocation = propertyFileLocation;
   }
```

Comment:
❓ Why does the `LdapClaimsHandler` care about the property file? Isn't the `PropertyFileClaimsHandler` responsible for this?

Reply:
This is a different properties file.